processor/compile/html-block.ts

```diff
@@ -1,9 +1,10 @@
 import { HTMLBlock } from '../../types';
+import { reformatHTML, getHProps } from '../utils'
 
 const htmlBlock = (node: HTMLBlock) => {
-  const html = node.data.hProperties.html;
-  const attributes = Object.keys(node.data?.hProperties).map(key => `${key}="${node.data?.hProperties[key]}"`).join(' ')
-  return `<HTMLBlock${attributes && ' ' + attributes}>{\`${ html }\`}</HTMLBlock>`;
+  const { runScripts, html } = getHProps<HTMLBlock['data']['hProperties']>(node);
+
+  return `<HTMLBlock${runScripts != null ? ' runScripts="' + runScripts + '"' : ''}>{\`\n${ reformatHTML(html) }\n\`}</HTMLBlock>`;
```

Comment:
Ahhh, so it looks like you're setting the `runScripts` prop to a string on compile. Maybe _that's_ the reason [it's sometimes coming in as a string, per my above question](https://github.com/readmeio/markdown/pull/906/files#r1642939558)? Can we update this to be a JSX-y value instead?

```diff
- `runScripts="${runScripts}"`
+ `runScripts={${runScripts}}`
```